src/store/ducks/lab/policy/sagas.ts

```diff
@@ -0,0 +1,91 @@
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import { AnyAction } from 'redux';
+
+import api from '../../../../services/api';
+import { AppState } from '../../..';
+import {
+  loadSuccess,
+  loadFailure,
+  createSuccess,
+  createFailure,
+  updateSuccess,
+  updateFailure,
+  deleteSuccess,
+  deleteFailure,
+} from './actions';
+import { LabPolicyTypes } from './types';
+
+const getToken = (state: AppState) => state.user.current.token;
+
+function* load(action: AnyAction) {
+  const { policyId, parameters } = action.payload;
+  const queryString = `/policies${policyId === 'all' ? '' : `/${policyId}`}`;
+  try {
+    const token: string = yield select(getToken);
+    const response: { [key: string]: any } = yield call(api.get, queryString, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      params: parameters,
+    });
+    yield put(loadSuccess(policyId, response.data));
+  } catch (err) {
+    yield put(loadFailure());
+  }
+}
+
+function* create(action: AnyAction) {
+  const body = action.payload;
+  try {
+    const token: string = yield select(getToken);
+    yield call(api.post, '/policies', {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      body,
+    });
+    yield put(createSuccess());
+  } catch (err) {
+    yield put(createFailure());
+  }
+}
+
+function* update(action: AnyAction) {
+  const { policyId, data } = action.payload;
+  try {
+    const token: string = yield select(getToken);
+    yield call(api.patch, `/policies/${policyId}`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      data,
+    });
```

Comment:
Same thing as above.
```suggestion
    yield call(api.patch, `/policies/${policyId}`, data, {
      headers: {
        Authorization: `Bearer ${token}`,
      },
    });
```